src/object_ref.jl

```diff
@@ -1,12 +1,60 @@
 mutable struct ObjectRef
-    oid::ray_jll.ObjectIDAllocated
+    # oid::ray_jll.ObjectIDAllocated
+    oid_hex::String
     owner_address::Union{ray_jll.AddressAllocated,Nothing}
```

Comment:
Should we keep the string form of this here instead?

Reply:
probably but I have those changes on later commits and dont' wanna have to cherry-pick em.  also, that didn't solve the problem `addr_proto.worker_id() != ""` failing